x-pack/legacy/plugins/graph/public/angular/templates/_graph.scss

```diff
@@ -22,6 +22,8 @@
  }
 
 .gphGraph__container {
+  display: flex;
```

Comment:
By making this a flex container, the svg grows to the correct size in Safari (otherwise it falls back to the default height of 150px)